feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -166,4 +162,32 @@ export class ConfiguratorStorefrontUtilsService {
       this.keyboardFocusService.set(key, group);
     }
   }
+
+  /**
+   * Change styling of element
+   *
+   * @param {string} querySelector - querySelector
```

Comment:
Please omit the type
```suggestion
   * @param  querySelector - querySelector
```